src/styles/index.ts

```diff
@@ -1331,6 +1331,18 @@ const styles = (theme: ThemeColors) =>
             lineHeight: variables.lineHeightNormal,
         },
 
+        textSupportingSmallSize: {
```

Comment:
Why not just use the existing classes for small font size, and for supporting/muted text color? This way we don't need to write a new style.